dnsimple/domains.go

```diff
@@ -25,6 +25,7 @@ type Domain struct {
 	AutoRenew    bool   `json:"auto_renew,omitempty"`
 	PrivateWhois bool   `json:"private_whois,omitempty"`
 	ExpiresOn    string `json:"expires_on,omitempty"`
```

Comment:
I wanted to make a deprecation warn in this attribute but couldn't find a good way on do it 🤔. any ideas?